ext/mysqli/mysqli.stub.php

```diff
@@ -110,111 +110,109 @@ public function connect(
         ?string $database = null,
         ?int $port = null,
         ?string $socket = null
-    ) {}
+    ): mysqli|null|false {}
```

Comment:
As far as I can tell, `mysqli` is not a possible return type here.

I think we should also adjust the implementation to return `true` on success rather than null. I think this was an unintentional artifact due to the mixing of the procedural and OO APIs.

Reply:
I've just tried to adjust the return types. The question is again what to change in 8.0... Are you ok with just removing the `mysqli` return type from the `connect()` method (without the rest of the changes)?

Reply:
Yes, for PHP 8.0 we can just omit the mysqli part.

Reply:
Actually, I think dropping the null return value from mysqli_connect should be fine as well.

Reply:
Now I understand - the `return;` cannot be reached in case of `mysqli_connenc()`. Nice.. Then I'll fix the return type again